File: src/com/basmilius/ps/bastools/framework/all/liveTemplates/AllLiveTemplatesProvider.java
package com.basmilius.ps.bastools.framework.all.liveTemplates;

import com.intellij.codeInsight.template.impl.DefaultLiveTemplatesProvider;

public class AllLiveTemplatesProvider implements DefaultLiveTemplatesProvider
{

	/**
	 * {@inheritDoc}
	 */
	@Override
	public final String[] getDefaultLiveTemplateFiles ()
	{
		return new String[]{"liveTemplates/MissingPhpStuffFramework"};
	}

	/**
	 * {@inheritDoc}
	 */
	@Override
	public final String[] getHiddenLiveTemplateFiles ()
	{
		return new String[0];
	}

}

File: src/com/basmilius/ps/bastools/framework/all/liveTemplates/AllLiveTemplatesProvider.kt
package com.basmilius.ps.bastools.framework.all.liveTemplates

import com.intellij.codeInsight.template.impl.DefaultLiveTemplatesProvider

/**
 * Class AllLiveTemplatesProvider
 *
 * @author Bas Milius
 * @package com.basmilius.ps.bastools.framework.all.liveTemplates
 */
class AllLiveTemplatesProvider: DefaultLiveTemplatesProvider
{

	/**
	 * {@inheritDoc}
	 *
	 * @author Bas Milius
	 */
	override fun getDefaultLiveTemplateFiles(): Array<String>
	{
		return arrayOf("liveTemplates/MissingPhpStuffFramework")
	}

	/**
	 * {@inheritDoc}
	 *
	 * @author Bas Milius
	 */
	override fun getHiddenLiveTemplateFiles(): Array<String?>
	{
		return arrayOfNulls(0)
	}

}
